packages/terra-form/src/components/Textarea.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { PropTypes } from 'react';
```

Comment:
Is this the file that was causing all the case sensitive issues? Should this be TextArea?